Formula/lib/libimobiledevice-glue.rb

```diff
@@ -0,0 +1,35 @@
+class LibimobiledeviceGlue < Formula
+  desc "Library with common system API code for libimobiledevice projects"
+  homepage "https://libimobiledevice.org/"
+  url "https://github.com/libimobiledevice/libimobiledevice-glue/releases/download/1.0.0/libimobiledevice-glue-1.0.0.tar.bz2"
+  sha256 "160a70e2edc318312fc40b6a71f85bfdfabdfba10bcfc5bb6fb40ed95088f4a0"
+  license "LGPL-2.1-or-later"
+  head "https://github.com/libimobiledevice/libimobiledevice-glue.git", branch: "master"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "libplist"
+
+  def install
+    system "./autogen.sh", *std_configure_args, "--disable-silent-rules" if build.head?
+    system "./configure", *std_configure_args, "--disable-silent-rules" if build.stable?
```

Comment:
```suggestion
    system "./configure", *std_configure_args, "--disable-silent-rules"
```

Reply:
This would mean configuring the project twice. `autogen.sh` runs `configure` once it has been generated.

Reply:
Alternatively, rather than using the archive from the releases tab on Github, we can just use the archive that Github generates, which would simplify the build process by only requiring to run `autogen.sh` on both stable and head builds.